book/coloring_and_theming.md

```diff
@@ -34,21 +28,27 @@ Here are the current options for `$env.config.table.mode`:
 - `none`
 - `other`
```

Comment:
`other` isn't valid. This is the list.
```nushell
❯ table --list                                                                                                              27  01:38:06 PM
╭────┬────────────────╮
│  0 │ basic          │
│  1 │ compact        │
│  2 │ compact_double │
│  3 │ default        │
│  4 │ heavy          │
│  5 │ light          │
│  6 │ none           │
│  7 │ reinforced     │
│  8 │ rounded        │
│  9 │ thin           │
│ 10 │ with_love      │
│ 11 │ psql           │
│ 12 │ markdown       │
│ 13 │ dots           │
│ 14 │ restructured   │
│ 15 │ ascii_rounded  │
│ 16 │ basic_compact  │
╰────┴────────────────╯
```

Reply:
I did not touch these. Can we handle the value updates separately?

I would prefer not to expand the scope of this format, text, and desc improvements and expansion with value doc updates/fixes.

Reply:
ok, i'm ok with incremental changes. it just needs to be done at some point because both lists are incomplete.

Reply:
I created https://github.com/nushell/nushell.github.io/pull/1774